go-controller/pkg/ovn/controller/services/utils.go

```diff
@@ -3,388 +3,116 @@ package services
 import (
 	"fmt"
 	"net"
+	"regexp"
 	"strings"
 
-	"github.com/pkg/errors"
-
-	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
-	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/ovn/gateway"
-	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/ovn/loadbalancer"
-	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+	globalconfig "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
```

Comment:
This makes sense but did you change it everywhere we use `"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"` just for the sake of consistency?

Reply:
I changed it many places, but changing it throughout all of ovn-k in a single PR is a recipe for hellish rebases.